src/views/HousingLottery/studentView/StudentApplicants/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Card, CardContent, CardHeader, Divider, Grid, TextField } from '@mui/material';
+import { useState } from 'react';
+import ApplicantFields from './ApplicantFields';
+import styles from '../../HousingLottery.module.css';
+
+const StudentApplicants = () => {
+  const initialApplicants = [
+    { firstName: '', lastName: '', email: '' },
+    { firstName: '', lastName: '', email: '' },
+    { firstName: '', lastName: '', email: '' },
+    { firstName: '', lastName: '', email: '' }
+  ];
+
+  const [applicants, setApplicants] = useState(initialApplicants);
+  const [emails, setEmails] = useState(initialApplicants.map(applicant => applicant.email));
+
+  const handleApplicantChange = (index, updatedApplicant) => {
+    const newApplicants = [...applicants];
+    newApplicants[index] = updatedApplicant;
+    setApplicants(newApplicants);
+
+    const newEmails = [...emails];
+    newEmails[index] = updatedApplicant.email;
+    setEmails(newEmails);
+  };
+
+  //   We will be having set number of applicants for the MVP and for now.
+  // const addApplicant = () => {
+  //   setApplicants([...applicants, { firstName: '', lastName: '', email: '' }]);
+  //   setEmails([...emails, '']);
+  // };
+
+  const handleSubmit = () => {
+    console.log(emails);
```

Comment:
You can keep this console log, but please remove all other console logs and commented-out code to make you files clean.

Reply:
Will do!!